tests/api/runtimes/test_serving.py

```diff
@@ -213,8 +193,8 @@ def test_mock_server_secrets(self, db: Session, client: TestClient):
         server = function.to_mock_server()
 
         # Verify all secrets are in the context
-        for secret_key in self.vault_secrets:
-            assert server.context.get_secret(secret_key) == self.vault_secret_value
+        # for secret_key in self.vault_secrets:
```

Comment:
some code in this file is commented out and some is deleted. I think we should do one or the other